.drone.yml

```diff
@@ -202,6 +202,9 @@ matrix:
     - PHP_VERSION: 7.2
```

Comment:
as commented by @patrickjahns - it is enough to run the code standard once with php5.6

Reply:
@patrickjahns please confirm that this is what you want.